src/components/App.js

```diff
@@ -4,6 +4,7 @@ import { a, div, h, h1, h2, nav } from 'react-hyperscript-helpers'
 import * as Nav from '../nav'
 import * as Style from '../style'
 import * as Utils from '../utils'
+import * as Config from './config'
```

Comment:
Suggest alphabetize so we don't have to think about order.

Reply:
So, this is the result of the alphabetizing by IntelliJ, which I believe is by path. I'm taking a stab at updating in my next PR.

Reply:
That's fine by me. I don't care what the convention is, just that I don't have to spend brain cycles on where to put things.